src/encode_task_qc_report.py

```diff
@@ -250,8 +240,7 @@ def str_ctl(i):
     'aligner': 'Aligner',
     'peak_caller': 'Peak caller',
     'genome': 'Genome',
-    'paired_end': 'Paired-end per replicate',
-    'ctl_paired_end': 'Control paired-end per replicate',
+    'read_end': 'Read end'
```

Comment:
I think this would be better entitled "Sequencing Endedness".  It isn't a real English word, but everyone knows what it means.  If you want something shorter just "Endedness" is fine, too.

Reply:
Ok, it's done.